framework/test/jest/unit/specs/modules/chain/synchronizer/block_synchronization_mechanism/peers.js

```diff
@@ -26,93 +26,81 @@ module.exports = {
 			blockVersion: 2,
 			maxHeightPrevoted: 1,
 			height: 66,
-			ip: '127.0.0.2',
-			wsPort: '5000',
```

Comment:
Is this supposed to be a `P2PPeerInfo`?
If so, would be better to just replace `ip` for `ipAddress` and leave `wsPort`

REF.: https://github.com/LiskHQ/lisk-sdk/blob/development/elements/lisk-p2p/src/p2p_types.ts#L73-L80